src/modules/assignment-4/questions/question4/Task4.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import "../Styles.css";
+
+const Task4 = () => {
+  const [formData, setFormData] = useState({});
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { password, confirmPassword } = formData;
+
+    // Password Validation
+    if (password !== confirmPassword) {
+      console.log("Password validation failed!");
+      return;
+    }
+
+    // Submitting the form
+    console.log("FormData =", formData);
+  };
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+
+    formData[name] = value;
+    setFormData({ ...formData });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div>
+        <label> Username </label>
+        <input
+          type="text"
+          id="username"
+          name="username"
+          onChange={handleChange}
+        />
+      </div>
```

Comment:
![image](https://github.com/ayush1-successive/react-training/assets/83696224/a5ae6095-3d74-451f-935a-b3fa653f6eac)
Show some error on ui when validation gets failed and success message when validation passes